### ISSUE TITLE
Hello ios11 system under the tableview cell pack up the keyboard page bounce phenomenon will appear

### ISSUE THREAD

**Issue by wangxiaodan820** [hackiftekhar/IQKeyboardManager#1091]
Hello, my view is a UItableview is added to the UIview hierarchy, and then again on the cell from the custom view the text box used to store UItextfile but click the middle fold after keyboard, tableview will automatically jump to the bottom, not return the original position  ，To ask how to solve the thank you

**Comment by wangxiaodan820** [comment]
这个问题 我又进一步跟踪  发现ios 11系统 键盘弹出可以自动顶起可能被遮盖的view，这个问题应该是系统问题

**Comment by SureshKumarD** [comment]
I'm also facing similar issue, In my case If I proceed to next scren(using pushViewController) while editing textfield(while keyboard is up). In the next screen I have a UICollectionView which is suddenly jump to bottom, although it's insets are (0,0,0,0). 

However, this is issue occurring only in iOS11.
I've applied contentInsetAdjustBehavior like below,
[myCollectionView setContentInsetAdjustmentBehavior:UIScrollViewContentInsetAdjustmentNever];
Still it's not solving my problem.

Any help would be really appreciated!!!
